Model/Transaction.php

```diff
@@ -141,6 +142,32 @@ public function getTransactionOrder()
         return $oOrder;
     }
 
+    /**
+     * Returns transaction's payment method name.
+     *
+     * @return string
+     *
+     * @since 1.3.0
+     */
+    public function getTransactionPaymentMethodName()
```

Comment:
is it returning the name as the return variable is called sPaymentId ?

Reply:
If there is a payment method with the given paymentId in the database, it returns the name of that payment method. Otherwise, it returns the paymentId itself (an example for this is "wiretransfer", coming as payment name for POI/PIA transaction in notification).

Reply:
Then this method is breaking the rule of the single responsibility as stated in the coding guidelines. Its also not what is described in the comment. Please make it generic split it up into methods so the code is reusable for isPoiPiaPaymentMethod(), as the second part of the method is the same.

Reply:
Done 👍